simplelistview/src/main/java/com/orhanobut/simplelistview/SimpleListView.java

```diff
@@ -119,6 +125,15 @@ public void onClick(View v) {
                     }
                 }
             });
+            if(itemLongClickListener != null){
```

Comment:
would be better to do same approach as itemClickListener. Currently if it is not set in the beginning, you cannot set it any longer. also could you do reformatting?